concent_api/gatekeeper/views.py

```diff
@@ -1,11 +1,41 @@
 from base64                         import b64decode
 
-from django.http                    import HttpResponse
+from django.conf                    import settings
+from django.http                    import JsonResponse, HttpResponse
 from django.views.decorators.http   import require_POST
 from django.views.decorators.csrf   import csrf_exempt
 
+from golem_messages.shortcuts       import load
+from golem_messages.message         import Message, init_messages
+
 
 @csrf_exempt
 @require_POST
 def file_transfer_auth(request):
-    return HttpResponse("", status=200)
+    init_messages()
```

Comment:
Why do you call `init_messages` here? `golem-messages` already calls this at module level. This should not be necessary.

Reply:
@Jakub89 What about this one?